package.json

```diff
@@ -25,6 +25,7 @@
     "@typegoose/typegoose": "^10.3.3",
     "env-var": "^7.3.0",
     "fastify": "^4.14.0",
+    "fastify-auth0-verify": "github:relaycorp/fastify-auth0-verify#auth0-agnostic-generated",
```

Comment:
Why are we forking this?

Reply:
I guess we will switch back to the original when the issue is merged?

Reply:
Both questions are answered in the PR description:

> Integrate fork of [fastify-auth0-verify](https://github.com/nearform/fastify-auth0-verify). See https://github.com/relaycorp/veraid-authority/issues/82.

Reply:
Yeah I saw that. checked the code first